File: GrowthWare_CS/GrowthWareFrameworkModel/Profiles/Base/MFormatter.cs
using System;
using System.Data;

namespace GrowthWare.Framework.Model.Profiles.Base
{
	public abstract class MFormatter : MProfile
	{
		protected string m_Body;

		protected override void Initialize(ref DataRow dr)
		{
			base.Initialize(ref dr);

			this.m_Body = base.GetString(ref dr, "Body");
		}
	}
}

File: GrowthWare_VB/GrowthWareFrameworkModel/Profiles/Base/MFormatter.vb
Imports System.Reflection
Imports GrowthWare.Framework.Model.Profiles.Base

Namespace ModelObjects.Base
	Public MustInherit Class MFormatter
		Inherits MProfile

		Protected m_Body As String

		Protected Overloads Sub Init(ByVal dr As DataRow)
			MyBase.Initialize(dr)
		End Sub
	End Class
End Namespace